### DISCUSSION TITLE
Exclude some industry standards like S3 from useNamingConvention

### DISCUSSION THREAD

**Discussion by hilja** [biomejs/biome#2014] [Ideas]
I completely agree that function params or config object keys should not start with a capital letter, but in S3 they do:

```js
S3.PutObject({
  Bucket: S3_IMAGES_BUCKET,
  Key,
  Body,
  ContentEncoding: 'base64',
  ContentType: img.mime,
  CacheControl: 'public, max-age=31536000', // year
})
```

Would it make sense to hard-code some exceptions for selected industry standards like S3? Or does it go against the Biome philosophy?

Related: https://github.com/biomejs/biome/discussions/1725

**Comment by ematipico** [comment]
Maybe it's worth continuing the discussion there in #1725. Opening a new discussion might fragment the topic too much. Just a thought.
